link-grammar/connectors.c

```diff
@@ -70,7 +70,7 @@ Connector * connector_new(const condesc_t *desc, Parse_Options opts)
 }
 
 /* ======================================================== */
-/* UNLIMITED-CONNECTORS handling. */
+/* Connector length limit handling. */
 
```

Comment:
Well, its in the dict as `UNLIMITED-CONNECTORS` so its nice to have it be searchable by that string name.   UI dunno. Maybe there is some other easy way to find this code.